### DISCUSSION TITLE
Error running setup.py build

### DISCUSSION THREAD

**Discussion by shoptechmedia** [acheong08/ChatGPT#480] [Q&A] [answered]
Hello, I'm trying to get this setup on my windows machine

Here's what I did:
First I cloned the repository, then I followed the instructions on how to install it
After that I tried running: python3 ./setup.py build

I keep getting this error:
https://prnt.sc/5-YHN43x1Sym

**Comment by acheong08** [comment, marked as answer]
Do `python3 -m pip install revChatGPT`

**Reply by shoptechmedia** [reply]
I still get the same issue. Do I do "python3 -m pip install revChatGPT" and "python3 ./setup.py build" on the same directory that I cloned the repository in?

**Reply by acheong08** [reply]
You don't need to clone the repository. Just use pip

**Reply by shoptechmedia** [reply]
alright. let me try doing this without cloning the repository

**Reply by shoptechmedia** [reply]
The reason I cloned it is that I cannot find where the "config.json" file is supposed to be after running "pip3 install --upgrade revChatGPT"

Is this a file that I have to create or does it come with the installation? Sorry about the questions. I'm quite new in this

**Reply by shoptechmedia** [reply]
Nevermind. I figured it out thanks :)
